src/SIM/SIMbase.C

```diff
@@ -2563,9 +2544,11 @@ bool SIMbase::injectPatchSolution (Vector& sol, const Vector& vec,
       nndof != getNoFields(basis) && this->getNoFields(2) > 0) {
     int key = basis << 16 + nndof;
     if (addMADOFs.find(key) == addMADOFs.end())
-      addMADOFs[key] = MADof(myModel, this->getNoNodes(true), basis, nndof);
+      addMADOFs[key] = std::move(setupAdditionalMADOF(myModel,
```

Comment:
Ser ikke forskjellen på på bruke std::move kontra bare = operatoren her. Antar det er for å spare en kopiering eller noe? Alternativet kunne være å gjøre om signaturen på setupAdditionalMADOF-metoden og kalle med addMADOFs[key] som [out] argument. Da er det noe klarere. Bortsett fra det er denne fixup-en i tråd med det jeg håpet på :-)